lambda/fulfillment/lib/middleware/lexRouter.js

```diff
@@ -277,13 +282,8 @@ async function processResponse(req, res, hook, msg) {
 
     const botResp = await handleRequest(req, res, hook, 'live');
     qnabot.log(`botResp: ${JSON.stringify(botResp, null, 2)}`);
-    let plainMessage = botResp.message;
-    let ssmlMessage;
-    // if messsage contains SSML tags, strip tags for plain text, but preserve tags for SSML
-    if (plainMessage?.includes('<speak>')) {
-        ssmlMessage = botResp.message;
-        plainMessage = plainMessage.replace(/<\/?[^>]+(>|$)/g, '');  // NOSONAR - javascript:S5852 - input is user controlled and we have a limit on the number of characters
-    }
+    let plainMessage = _.get(botResp, 'messages.PlainText', '');
```

Comment:
Need to confirm that the messages.Plaintext and messages.ssml do return the plaintext and ssml message separately now